### ISSUE TITLE
Updating plugins doesn't work on Windows

### ISSUE THREAD

**Issue by pax0707** [oznu/homebridge-config-ui-x#53]
Updating plugins doesn't work from web, works from cmd.

```
[2018-3-12 21:46:47] [Config] Running Command: C:\Program Files\nodejs\npm.cmd install --unsafe-perm homebridge-config-ui-x@latest
[2018-3-12 21:47:31] [Config] Running Command: node C:\Users\pax07\node_modules\nsp\bin\nsp check
[2018-3-12 21:47:32] [Config] Error: File not found:
    at new WindowsPtyAgent (C:\Users\pax07\node_modules\node-pty\src\windowsPtyAgent.ts:52:22)
    at new WindowsTerminal (C:\Users\pax07\node_modules\node-pty\src\windowsTerminal.ts:50:19)
    at Object.spawn (C:\Users\pax07\node_modules\node-pty\src\index.ts:30:10)
    at Bluebird (C:\Users\pax07\node_modules\homebridge-config-ui-x\src\pm.ts:182:24)
    at Promise._execute (C:\Users\pax07\node_modules\bluebird\js\release\debuggability.js:303:9)
    at Promise._resolveFromExecutor (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:483:18)
    at new Promise (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:79:10)
    at PackageManager.runNspScan (C:\Users\pax07\node_modules\homebridge-config-ui-x\src\pm.ts:170:12)
    at executeCommand.then (C:\Users\pax07\node_modules\homebridge-config-ui-x\src\pm.ts:434:28)
    at tryCatcher (C:\Users\pax07\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:638:18)
    at Promise._resolveCallback (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:432:57)
    at Promise._settlePromiseFromHandler (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:524:17)
    at Promise._settlePromise (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:693:18)
    at Promise._fulfill (C:\Users\pax07\node_modules\bluebird\js\release\promise.js:638:18)
    at MappingPromiseArray.PromiseArray._resolve (C:\Users\pax07\node_modules\bluebird\js\release\promise_array.js:126:19)
[2018-3-12 21:49:33] Got SIGINT, shutting down Homebridge...
Terminate batch job (Y/N)? y

C:\Users\pax07>npm i homebridge-config-ui-x
npm WARN saveError ENOENT: no such file or directory, open 'C:\Users\pax07\package.json'
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\pax07\package.json'
npm WARN pax07 No description
npm WARN pax07 No repository field.
npm WARN pax07 No README data
npm WARN pax07 No license field.

+ homebridge-config-ui-x@3.2.0
updated 1 package in 12.731s

C:\Users\pax07>
```

**Comment by oznu** [comment]
This will only effect windows. I can spot the problem and will push up a fix later.

**Comment by oznu** [comment]
Fixed in 3.2.1. You can upgrade via the web ui, just ignore the error at the end, the plugin still gets updated, then restart the server and do full refresh of the page (CTRL+F5) and everything should work next time.
